examples/musicPlayer/src/3_useMediaPlayer.tsx

```diff
@@ -0,0 +1,108 @@
+import { usePlayMedia } from "@/lib/audio/usePlayMedia";
+import { usePlayState } from "@/lib/audio/usePlayState";
+import { useAccount, useCoState } from "@/lib/jazz";
+import { MusicTrack, Playlist } from "@/1_schema";
+import { BinaryCoStream } from "jazz-tools";
+import { useEffect } from "react";
+
+export function useMediaPlayer() {
+    const { me } = useAccount();
```

Comment:
I have some issues with this hook when using the `depth` param.

By calling the hook like this when trying a cold start the account never gets loaded:
```
const { me } = useAccount({
       root: {
            activePlaylist: {
                tracks: [{}],
            },
            activeTrack: {
                file: [],
            },
        },
});
```

Also it was causing some type errors on the below assignements:

```ts
me.root.activeTrack = next;
```

@aeplay is this a wrong usage of the API or a bug to report?